projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -22,24 +22,25 @@
   - projects/control-service/projects/helm_charts/pipelines-control-service/**/*
 
 .control_service_base_build:
-  image: docker:19.03.15
+  image: docker:20.10.21
   variables:
```

Comment:
The pipeline seems to fail during:
```
[32;1m$ pip3 install --upgrade pip && pip3 install awscli[0;m
Traceback (most recent call last):
  File "/usr/bin/pip3", line 33, in <module>
    sys.exit(load_entry_point('pip==22.3.1', 'console_scripts', 'pip3')())
  File "/usr/bin/pip3", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 969, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 548, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for pip
```
In comparison to a successful one that looks like:
```
32;1m$ pip3 install --upgrade pip && pip3 install awscli[0;m
Looking in indexes: https://pypi.org/simple, https://pypi.org/simple
Requirement already satisfied: pip in /usr/lib/python3.8/site-packages (20.3.4)
Collecting pip
  Downloading pip-22.3.1-py3-none-any.whl (2.1 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.3.4
    Uninstalling pip-20.3.4:
      Successfully uninstalled pip-20.3.4
Successfully installed pip-22.3.1
```

I would suggest to test if unblocked by some of the solutions mentioned in  https://stackoverflow.com/questions/54552367/pip-cannot-find-metadata-file-environmenterror
For example force reinstall

Reply:
Done